ee/clickhouse/models/test/test_property.py

```diff
@@ -423,6 +423,7 @@ def test_prop_event_denormalized_ints(self):
         self.assertEqual(len(self._run_query(filter)), 1)
 
 
+@pytest.mark.django_db
```

Comment:
context?

Reply:
This PR stores this setting in a django db model so the test needs access to the db to grab whether or not materialized columns are enabled!